docs/CHANGELOG.md

```diff
@@ -3,6 +3,10 @@
 
 ## 1.0
 
+### 1.3.1
+
```

Comment:
I would bump it into 1.4.0. This is not teeny-tiny change

Reply:
I think you're right!